packages/components/src/checkbox-control/index.js

```diff
@@ -27,6 +37,7 @@ function CheckboxControl( { label, className, heading, checked, help, instanceId
 			<label className="components-checkbox-control__label" htmlFor={ id }>
 				{ label }
 			</label>
+			{ children }
```

Comment:
Not sure about this change. None of the other control components have `children` as a prop.